workloads/files/workload-http-script-cm.yml

```diff
@@ -0,0 +1,1218 @@
+      echo "" > /var/lib/pbench-agent/tools-default/perf
+      echo "" > /var/lib/pbench-agent/tools-default/pidstat
+      master_nodes=`oc get nodes --show-labels -l pbench_agent=true,node-role.kubernetes.io/master= | grep master | awk '{print $1}'`
+      for node in $master_nodes; do
+        echo "master" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+      infra_nodes=`oc get nodes --show-labels -l pbench_agent=true,node-role.kubernetes.io/infra= | grep infra | awk '{print $1}'`
+      for node in $infra_nodes; do
+        echo "infra" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+      worker_nodes=`oc get nodes --show-labels -l pbench_agent=true,node-role.kubernetes.io/worker= | grep worker | awk '{print $1}'`
+      for node in $worker_nodes; do
+        echo "worker" > /var/lib/pbench-agent/tools-default/remote@$node
+      done
+    fi
+    source /opt/pbench-agent/profile
+    # End pbench Configuration
+
+    # Test Configuration
+    echo "$(date -u) Configuring HTTP test"
+    test_dir=/tmp/http-ci-tests
+    mkdir -p $test_dir/content/quickstarts/{nginx,stress}
+    install -m 644 /root/workload/server-*.yaml $test_dir/content/quickstarts/nginx
+    install -m 644 /root/workload/stress-*.yaml $test_dir/content/quickstarts/stress
+    install -m 755 /root/workload/*.sh $test_dir
+    export SERVER_RESULTS=${SERVER_RESULTS:-scp://root@${OCP_NODE_NAME}:/var/lib/pbench-agent}
+    #export SERVER_RESULTS=${SERVER_RESULTS:-kcp://${OCP_POD_NAME}:/var/lib/pbench-agent}
+    #echo "$(date -u) Test results will be copied to: $SERVER_RESULTS"
+    # End Test Configuration
+
+    # Start of Test Code
+    echo "$(date -u) Running HTTP test"
+    cd $test_dir
+    ./http-test.sh all
+    sleep inf
+    echo "$(date -u) Completed HTTP test"
+    # End of Test Code
+
+  ###################################################################################################
+  env.sh: |
+    # Test configuration: a label to append to pbench directories and mb result directories.
+    export TEST_CFG=${TEST_CFG:-1router}
+    # A list of paths to k8s configuration files.  Colon-delimited for Linux.
+    export KUBECONFIG=${KUBECONFIG:-/root/.kube/config}
+    # Use benchmarking and performance analysis framework Pbench.
+    export PBENCH_USE=${PBENCH_USE:-false}
+    # Pbench scraper/wedge usage for r2r analysis.
+    export PBENCH_SCRAPER_USE=${PBENCH_SCRAPER_USE:-false}
+    # Clear results from previous pbench runs.
+    export CLEAR_RESULTS=${CLEAR_RESULTS:-true}
+    # Move the benchmark results to a pbench server.
+    export MOVE_RESULTS=${MOVE_RESULTS:-false}
+    # HTTP-test specifics
+    # Endpoint for collecting results from workload generator node(s).  Keep empty if copying is not required.
+    # - scp://[user@]server:[path]
+    # - kcp://[namespace/]pod:[path]
+    # Note: when PBENCH_USE=true, path is overriden by benchmark_run_dir environment variable
+    export SERVER_RESULTS=${SERVER_RESULTS}
```

Comment:
I believe the issue above might have been caused by the container running on a worker node, rather than the pbench node and not having the ssh keys configured properly for the results to be copied back.  In any case, perhaps using kcp:// by default and `OCP_POD_NAME` will be a safer option for this framework.  Need to think about this.